### ISSUE TITLE
Thrusters and gyros can flicker between on/off rapidly

### ISSUE THREAD

**Issue by mirrorcult** [space-wizards/space-station-14#27754]
## Description

flickers between ticks looks weird as hell

**Reproduction**

1) take eshuttle
2) ghost
3) (?) profit idk if theres more steps but ive seen this a few times

**Screenshots**

![SS14 Loader_2024-05-06_20-55-10](https://github.com/space-wizards/space-station-14/assets/19853115/0327964e-be87-485e-8c56-15bfe12de1bb)

**Additional context**

**Comment by metalgearsloth** [comment]
Yeah making it dynamically take power was a terrible idea, need to revert https://github.com/space-wizards/space-station-14/pull/26690

**Comment by lzk228** [comment]
the thing was reverted, so i suppose it shouldn't be an issue
